### PR TITLE
Boundary

### PR DESCRIPTION
**Pull request by kylrbutlr**
There's not much here to review but let's double check just to make sure.

### CODE REVIEW

File: Scripts/boundary.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import numpy as np
+from scipy.spatial import Delaunay
+from shapely.geometry import Polygon
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+fig = plt.figure()
```
Comment (brianvu155) [inline Scripts/boundary.py]: Put this in main so its not hard to find (unless this is standard in python)

File: Scripts/boundary.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        circum_r = a * b * c / (4.0 * area)
+        if circum_r < alpha:
+            add_edge(edges, ia, ib)
+            add_edge(edges, ib, ic)
+            add_edge(edges, ic, ia)
+    return edges
+
+
+if __name__ == "__main__":
+    test_file = r'C:\Users\Kyler\Documents\GitHub\CSE450Project\capstone\67P_test_scan.csv'
```
Comment (brianvu155) [inline Scripts/boundary.py]: comment where people can find this file

File: Scripts/boundary.py (hunk 1 above)
Comment (kylrbutlr) [reply Scripts/boundary.py]: Lol actually I forgot to remove those since we don't need them.

File: Scripts/boundary.py (hunk 2 above)
Comment (kylrbutlr) [reply Scripts/boundary.py]: I just added it to the Test folders and created a relative path.

**Pull request merged**